modules/cudaimgproc/include/opencv2/cudaimgproc.hpp

```diff
@@ -145,7 +145,7 @@ E.g. Given an RGBA image, aDstOrder = [3,2,1,0] converts this to ABGR channel or
 
 The methods support arbitrary permutations of the original channels, including replication.
  */
-CV_EXPORTS_W void swapChannels(InputOutputArray image, const int dstOrder[4], Stream& stream = Stream::Null());
+CV_EXPORTS void swapChannels(InputOutputArray image, const int dstOrder[4], Stream& stream = Stream::Null());
 
```

Comment:
Unfortunately, the generator cannot handle arrays in the form of `int dstOrder[4]` properly, thus undo the wrapping..
This bad behavior needs to be addressed, hopefully in the near future.